Expensify/App
# Refactor workspace settings

This PR adds a lot of feature promotion to the workspace settings

### Fixed Issues
$ https://github.com/Expensify/App/issues/5689

### Tests
1. Log into NewDot
2. Create a workspace
3. Go to Settings > Workspace
4. Navigate around all the pages and see that most features are locked without a bank account
5. Add a bank account
6. Navigate around all the pages and see that the features are now unlocked

### QA Steps
Tests are all written [in this doc](https://docs.google.com/document/d/1aTSCuwT_LS0W9w9svzgSWThflHRNOdxAbBhhgk9x530/edit#heading=h.nhtsmauf72jq) and will be performed internally

### Tested On


### Screenshots


#### Web
![image](https://user-images.githubusercontent.com/1228807/136459057-ea7ecb19-5ea3-4c6e-819c-3b7653512721.png)


#### Mobile Web
![image](https://user-images.githubusercontent.com/1228807/136459760-8dc8ee5d-0f17-4a17-855e-514889a30ace.png)


#### Desktop
![image](https://user-images.githubusercontent.com/1228807/136459030-a8488114-dbb3-4d8a-8738-cd8782715b50.png)


#### iOS
![image](https://user-images.githubusercontent.com/1228807/136458554-0fd45709-593e-41aa-9ed5-cfda1b6dca09.png)

![image](https://user-images.githubusercontent.com/1228807/136458566-fa2a1b24-6457-4e36-b9d9-0c3e9967a100.png)


#### Android
